src/platform/webengine/BlinkWebViewProfileHelper.cpp

```diff
@@ -66,6 +73,16 @@ int BlinkWebViewProfileHelper::maskForBrowsingDataType(const char* type) {
     return 0;
 }
 
-void BlinkWebViewProfileHelper::setProxyRules(const std::string& proxy_rules) {
-    webos::WebViewProfile::GetDefaultProfile()->SetProxyRules(proxy_rules);
+webos::WebViewProfile* BlinkWebViewProfileHelper::getProfile(const std::string& app_id) {
+    if (app_profile_map_.find(app_id) == app_profile_map_.end())
+       return nullptr;
+    return app_profile_map_[app_id];
 }
+
+void BlinkWebViewProfileHelper::buildProfile(const std::string& app_id, const std::string& proxy_rules)
```

Comment:
For a next patch, you need to delete the profile entry when the app is closed. Could you create a jira ticket for tracking that work?